pkg/scannerv4/mappers/mappers.go

```diff
@@ -695,6 +770,47 @@ func pkgFixedBy(enrichments map[string][]json.RawMessage) (map[string]string, er
 	return pkgFixedBys, nil
 }
 
+// cveEPSS unmarshals and returns the EPSS enrichment, if it exists.
+func cveEPSS(enrichments map[string][]json.RawMessage, ctx context.Context) (map[string]epssDetail, error) {
+	enrichmentList := enrichments[epss.Type]
+	if len(enrichmentList) == 0 {
+		zlog.Warn(ctx).
+			Str("enrichments:", epss.Type).
+			Msg("no EPSS enrichments found")
+		return nil, nil
+	}
+
+	var epssMap map[string][]epssDetail
+	err := json.Unmarshal(enrichmentList[0], &epssMap)
+	if err != nil {
+		return nil, fmt.Errorf("error unmarshaling EPSS enrichment: %w", err)
+	}
+
+	if len(epssMap) == 0 {
+		return nil, nil
+	}
+
+	res := make(map[string]epssDetail)
+	for k, details := range epssMap {
+		if len(details) == 0 {
+			continue
+		}
+
+		// assume the first element is the relevant EPSS detail
+		detail := details[0]
+		if detail.CVE == "" {
+			zlog.Warn(ctx).
+				Str("package:", k).
+				Msg("no CVE linked to an EPSS score")
+			continue
+		}
```

Comment:
How often does this happen?

Reply:
I don't think this is possible since the enricher parses the raw data for each field